app/components/request_table_row_component.html.erb

```diff
@@ -1,6 +1,13 @@
 <tr>
   <td><%= link_to_record %></td>
   <td><%= @request_details.pickupServicePoint %></td>
-  <td><%= @request_details.patronComments %></td>
+  <td>
+    <p><%= @request_details.patronComments %></p>
+    <%- if @request_details.cancellationReason.present? %>
+      <p><%= @request_details.cancellationReason %></p>
+      <p><%= @request_details.cancellationComment %></p>
```

Comment:
I don't know if cancellation comment will always be present, it will depend on whether the circulation person writes something. I assume they would, but who knows

Reply:
Fair point. I've added a check for it. Ta!